src/toolchain/web-guide.md

```diff
@@ -0,0 +1,147 @@
+<!--
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+
+# Intro to Web Builds
+
+Web builds are a fair bit more difficult to get started with compared to native builds.
+This will be a complete guide on how to get things compiled.
+
+Information to consider before starting:
+
+- Setting up a web server to host and share your game is considered out of scope of this guide, and is best explained elsewhere.
+- Web support with `gdext` is experimental and should be understood as such before proceeding.
+
+---
+
+
+## Installing Pre-Requisites
+
+- Install a nightly build of `rustc`, the `wasm32-unknown-emscripten` target for `rustc`, and `rust-src`.
+Assuming that Rust was installed with `rustup`, this is quite simple.
+
+  ```sh
+  rustup toolchain install nightly
+  rustup component add rust-src --toolchain nightly
+  rustup target add wasm32-unknown-emscripten --toolchain nightly
+  ```
+
+- Install Emscripten
+  - The simplest way to achieve this is to install `emsdk` from the git repo. Recommended to use version 3.1.39 for now[^1]
+
+  ```sh
+  git clone https://github.com/emscripten-core/emsdk.git
+  cd emsdk
+  ./emsdk install 3.1.39
+  ./emsdk activate 3.1.39
+  source ./emsdk.sh     (or ./emsdk.bat on windows)
+  ```
+
+  - It would also be HIGHLY recommended to follow the instructions in the terminal to add `emcc`[^2] to your `PATH`.
+    If not, it is necessary to manually `source` the `emsdk.sh` file in every new terminal prior to compilation.
+    This is platform-specific.
+
+---
+
+
+## Project Configuration
+
+- Enable the `experimental-wasm` feature on `gdext` in the `Cargo.toml` file.
+  It is also strongly recommended to enable the `lazy-function-tables` feature to avoid an extremely long compile time with release builds.
+  It is likely to due a bug, but is still being investigated.  Edit the line to something like the following:
+
+  ```toml
+  godot = { git = "https://github.com/godot-rust/gdext", branch = "master", features = ["experimental-wasm", "lazy-function-tables"] }
+  ```
+
+- At the same level as your `Cargo.toml`, create a `.config` directory and inside of that,
+  create a `config.toml` file (if one doesn't already exist). The contents of this file should be as follows
+
+  ```toml
+  [target.wasm32-unknown-emscripten]
+  rustflags = [
+      "-C", "link-args=-sSIDE_MODULE=2",
+      "-C", "link-args=-sUSE_PTHREADS=1",
+      "-C", "target-feature=+atomics,+bulk-memory,+mutable-globals",
+      "-Zlink-native-libraries=no"
+  ]
+  ```
+
+- Edit the project's `.gdextension` file to include support for web exports.
+  This file will probably be at `godot/EXTENSIONNAME.gdextension`.
```

Comment:
```suggestion
  This file will probably be at `godot/{YourCrate}.gdextension`.
```
to be consistent with [Hello World](https://godot-rust.github.io/book/intro/hello-world.html).